robot_editor/Config[] sample_points
